ksql-engine/src/test/java/io/confluent/ksql/KsqlEngineTest.java

```diff
@@ -108,15 +109,14 @@
 
   private KsqlEngine ksqlEngine;
   private ServiceContext serviceContext;
-  private FakeKafkaTopicClient topicClient;
+  @Spy
```

Comment:
Cool! Learning more Mockito with each PR review :)

Reply:
lol. Happy to help.